apps/www/src/state/plugin/viewCollection.js

```diff
@@ -1,28 +1,77 @@
-import { hydra, schema } from '@tpluscode/rdf-ns-builders'
-import { newReference } from '../../clownface.js'
+import { hydra, rdf, schema } from '@tpluscode/rdf-ns-builders'
+import * as ns from '../../ns.js'
 
 export const viewCollection = {
   model: {
+    state: {},
+    reducers: {
+      setPointer(state, pointer) {
+        return { ...state, pointer }
+      },
+      setNewViewOperation(state, newViewOperation) {
+        return { ...state, newViewOperation }
+      },
+      setNewViewShape(state, newViewShape) {
+        return { ...state, newViewShape }
+      },
+    },
     effects(store) {
       const dispatch = store.getDispatch()
 
       return {
-        async deleteView(pointer) {
-          const { resource, core } = store.getState()
+        'core/setContentResource': ({ pointer }) => {
+          if (pointer.has(rdf.type, ns.viewBuilder.ViewCollection).term) {
+            dispatch.viewCollection.setPointer(pointer)
+            dispatch.viewCollection.loadNewViewShape()
+          }
```

Comment:
Initialises the view collection state when an instance of `ViewCollection` is loaded to core state